src/components/explore-section/Circuit/ListView/ResizableTitle.tsx

```diff
@@ -0,0 +1,49 @@
+import { Resizable, ResizeCallbackData } from 'react-resizable';
+
+export type ResizableTitleProps = {
+  onResize?: (e: React.SyntheticEvent, data: ResizeCallbackData) => void;
+  width?: number;
+  [key: string]: any;
+};
+
+export default function ResizableTitle(props: ResizableTitleProps) {
+  const { onResize, width, ...restProps } = props;
+
+  if (!width) {
+    // eslint-disable-next-line react/jsx-props-no-spreading
+    return <th {...restProps} />;
+  }
+
+  const handleKeyDown = (e: React.KeyboardEvent<HTMLSpanElement>) => {
+    if (e.key === 'Enter' || e.key === ' ') {
+      e.stopPropagation();
+    }
+  };
+
+  const handleResizeEvent = (e: React.SyntheticEvent, data: ResizeCallbackData) => {
+    if (onResize) {
+      onResize(e, data);
+    }
+  };
+
+  return (
+    <Resizable
```

Comment:
I've tried locally and it doesn't seem to be working, not sure I did it right, but please see the screencast:

https://github.com/user-attachments/assets/9c8a3e2e-4495-40f5-8574-718e8af35f3b